freemocap/fmc_mediapipe_annotation.py

```diff
@@ -0,0 +1,99 @@
+                                connection_drawing_spec=mp_drawing_styles
+                                .get_default_face_mesh_contours_style())
+        mp_drawing.draw_landmarks(image=image,
+                                landmark_list=mediapipe_results.face_landmarks,
+                                #   connections=mp_holistic.FACEMESH_TESSELATION,
+                                connections=mp_holistic.FACEMESH_CONTOURS,
+                                landmark_drawing_spec=None,
+                                connection_drawing_spec=mp_drawing_styles
+                                .get_default_face_mesh_tesselation_style())
+        mp_drawing.draw_landmarks(image=image,
+                                landmark_list=mediapipe_results.pose_landmarks,
+                                connections=mp_holistic.POSE_CONNECTIONS,
+                                landmark_drawing_spec=mp_drawing_styles
+                                .get_default_pose_landmarks_style())
+        mp_drawing.draw_landmarks(
+                            image=image,
+                            landmark_list=mediapipe_results.left_hand_landmarks,
+                            connections=mp_holistic.HAND_CONNECTIONS,
+                            landmark_drawing_spec=None,
+                            connection_drawing_spec=mp_drawing_styles
+                            .get_default_hand_connections_style())
+        mp_drawing.draw_landmarks(
+                            image=image,
+                            landmark_list=mediapipe_results.right_hand_landmarks,
+                            connections=mp_holistic.HAND_CONNECTIONS,
+                            landmark_drawing_spec = None,
+                            connection_drawing_spec=mp_drawing_styles
+                            .get_default_hand_connections_style())
+        return image
+
+    drawing_spec = mp_drawing.DrawingSpec(thickness=1, circle_radius=1)
+
+    cap = cv2.VideoCapture(str(synced_video_path))
+    
+    video_frame_width = int(cap.get(cv2.CAP_PROP_FRAME_WIDTH))
+    video_frame_height = int(cap.get(cv2.CAP_PROP_FRAME_HEIGHT))
+    framerate = cap.get(cv2.CAP_PROP_FPS)
+
+    writer = cv2.VideoWriter(str(path_to_save_video/name_for_saved_video),
+                            cv2.VideoWriter_fourcc(*'mp4v'),
+                            framerate,
+                            (video_frame_width, video_frame_height))
+
+    with mp_holistic.Holistic(
+        min_detection_confidence=0.5,
+        min_tracking_confidence=0.5,
+        model_complexity=2) as holistic:
+        success = True
+        while success:
+            success, raw_image = cap.read()
+            if not success:
+                break
+
+            mediapipe_results = holistic.process(raw_image)
+            annotated_image = annotate_image_with_mediapipe_data(raw_image, mediapipe_results)
+
+            writer.write(annotated_image)
+
+    writer.release()
+    cap.release()
+
```

Comment:
I've started trying to get in the habit of adding an example of how to use these functions when writing code like this. 

i.e. via  _ 

```python
if __name__ == "__main__":
    example_of_how_to_use_the_functions_in_this_script.run()
    
```

Obviously that's tricky to do in our current method, because there's a lot of machinery to generate the `session` object that's hard to reproduce here 

(but mostly I just wrote this because I  wanted to see what it was like to put a comment on your code :P )